shuttle/templates/loaded_views/user_modal.html

```diff
@@ -80,4 +129,13 @@ <h5 class="modal-title">Change role of {{ username }}</h5>
                 }
             })
         }
+
+        function adminButton(){
+            document.getElementById("role-dropdown").innerText = "Administrator"
+            role = "Administrator"
+        }
+        function driverButton(){
+            document.getElementById("role-dropdown").innerText = "Driver"
+            role = "Driver";
+        }
```

Comment:
Instead of having two functions just combine into one that passes over the role in case we add additional roles in the future.